dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -80,3 +80,12 @@ preferCRLs.desc=By default, OCSP is the preferred mechanism for checking revocat
 
 disableFallback=No fallback
 disableFallback.desc=Disable the certificate revocation status checker fallback mechanism. During certificate revocation status check, the default behavior is fallback and check by using the non-preferred mechanism (OCSP or CRLs). Default is false.
+
+certCheckerSchedule=Certificate checker schedule
+certCheckerSchedule.desc=Interval for the ACME CA service to check if the certificate is expiring or revoked. If expiring or revoked, the certificate is renewed. If the first renew request fails, the certificate renew request continues according to the timing set by the certCheckerErrorSchedule attribute until a new certificate is received. Setting the certCheckerSchedule property to zero or a negative value disables automatic certificate renew requests.
+
+certCheckerErrorSchedule=Certificate checker error schedule
+certCheckerErrorSchedule.desc=Interval for the ACME CA service to check if the certificate is expiring or revoked after the first renew request failed using the certCheckerSchedule property interval. The certCheckerErrorSchedule attribute can be set to a shorter interval than the certCheckerSchedule attribute. The interval from the certCheckerSchedule attribute is resumed after the certificate is renewed. 
```

Comment:
`Interval for the ACME CA service to check if the certificate is expiring or revoked after the first renew request failed using the certCheckerSchedule property interval`

This is a little confusing- does it mean that this is interval at which the service checks for an expiring or revoked certificate _after_ a failed renew request? If so, why? If there's been a renew request, then hasn't the service already determined that the certificate is expiring or revoked? Or maybe if the certCheckerSchedule detects an "expiring" certificate, this checks to make sure it hasn't actually expired since the failed renew request? Maybe I'm just reading it wrong

Reply:
I rewrote it -- does that help? Basically, we have a background thread that wakes up on an interval defined by certCheckerSchedule. If the certificate is expiring or revoked, we request a new one. If that request fails, we'll change the interval to be certCheckerErrorSchedule. So normal schedule is daily, but if we need a new cert, we can swap to hourly so we hopefully get a new certificate in time/faster.